vector/src/main/java/im/vector/app/features/spaces/explore/SpaceDirectoryFragment.kt

```diff
@@ -60,30 +68,48 @@ class SpaceDirectoryFragment @Inject constructor(
         }
         epoxyController.listener = this
         views.roomDirectoryPickerList.configureWith(epoxyController)
+
+        childFragmentManager.addFragmentOnAttachListener(fragmentOnAttachListener)
     }
 
     override fun onDestroyView() {
+        childFragmentManager.removeFragmentOnAttachListener(fragmentOnAttachListener)
```

Comment:
is `fragment.interactionListener` set to null? I'm not sure

Reply:
made the listener as weak reference in the bottomsheet dialog